.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+
+branches:
+  only:
+    - master
```

Comment:
why not all?

Reply:
It's to avoid 'double builds on pull requests': https://docs.travis-ci.com/user/pull-requests#double-builds-on-pull-requests

branches.only controls builds in response to merges. Builds in response to pull requests are set in Travis CI settings.